walletnode/services/artworksearch/helper.go

```diff
@@ -0,0 +1,31 @@
+package artworksearch
+
+import (
+	b64 "encoding/base64"
+	"encoding/json"
+	"fmt"
+)
+
+func inIntRange(val int, min *int, max *int) bool {
```

Comment:
please don't use pointer if no change for it in function

Reply:
these vals can be nil.. you can see nil checks inside func bodies. 
this is intended.